packages/monitoring/package.json

```diff
@@ -20,7 +20,7 @@
     "node": "20 || 22"
   },
   "scripts": {
-    "build": "rollup -c rollup.config.ts --bundleConfigAsCjs",
+    "build": "tsc && rollup -c rollup.config.ts --bundleConfigAsCjs",
```

Comment:
maybe type checking is more of a _test_ action? (which is a prerequisite of publishing a release so would be certain to happen in CI)

Reply:
Can do, it's weird as `build` is more of a "produce artifact" task rather than "build the product"

Reply:
that's done now

Reply:
would linting also fit in to that too?

Reply:
CI does `npm test` and `npm run lint` in parallel for all runs. but not `npm run build`; that only happens on release

Reply:
I can fix that after a release

Reply:
how about this? https://github.com/ministryofjustice/hmpps-typescript-lib/pull/25